tests/test_base.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import, division, print_function
+import unittest
+
+import numpy as np
+from numpy.testing import assert_equal
+
+import astshim as ast
+from astshim.test import ObjectTestCase
+
+
+class TestObject(ObjectTestCase):
+
+    def test_arrayFromVector(self):
+        nAxes = 3
+        nValues = 5
+        dataVec = np.random.rand(nAxes * nValues)
```

Comment:
Does this use a fixed seed?

Reply:
Good catch. In this case I don't see how it can make any difference, but providing a seed is best practice.